akvo/rsr/models/iati_import_log.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..fields import ValidXMLTextField
+
+SEVERITY_CODE = {
+    0: _(u'information'),
+    1: _(u'critical error'),
+    2: _(u'value not saved'),
+    3: _(u'value partly saved'),
+}
+
+
+class IatiImportLog(models.Model):
+    iati_import = models.ForeignKey(
+        'IatiImport', verbose_name=_(u'iati_import'), related_name='iati_import_logs'
+    )
+    project = models.ForeignKey(
+        'Project', verbose_name=_(u'project'), related_name='iati_project_import_logs',
+        blank=True, null=True
+    )
+    severity = models.IntegerField(_(u'severity'), default=1)
```

Comment:
I think this could be a PositiveSmallIntegerField with a choices attribute that limits the possible values to the severities.